src/Views/Form/Themes/Sequoia/sections/introduction.php

```diff
@@ -1,16 +1,17 @@
 <?php
-$form_title = sprintf(
-	'<h2 class="give-form-title">%1$s</h2>',
-	get_the_title( $form->ID )
-);
+
+// Get headline and description
+$headline    = $this->theme_options['introduction']['headline'];
```

Comment:
@henryholtgeerts in `src` folder for `psr-4` coding standard, so property name must be in camel case

Reply:
fixed: https://github.com/impress-org/givewp/compare/b1375fdefbcc...6d7c638d8365